card/mod/01_core/set/all/trash.rb

```diff
@@ -44,13 +44,13 @@ def delete!
   end
 end
 
-event :validate_delete_children, :validate, on: :delete do
+event :validate_delete_children, :prepare_to_validate, on: :delete do
   children.each do |child|
     child.trash = true
     add_subcard child
-    next if child.valid?
-    child.errors.each do |field, message|
-      errors.add field, "can't delete #{child.name}: #{message}"
-    end
+    # next if child.valid?
+    # child.errors.each do |field, message|
+    #   errors.add field, "can't delete #{child.name}: #{message}"
+    # end
```

Comment:
`child.valid?` means we mess with the order of the stages. child is added as subcard so it will be validated anyway and 'handle_subcard_errors' will take care of possible errors. We loose the "can't delete" in the error message at the moment. Not sure if the message will become less clear.